soh/soh/SohGui/SohMenuEnhancements.cpp

```diff
@@ -1123,6 +1123,9 @@ void SohMenu::AddMenuEnhancements() {
         .Options(CheckboxOptions().Tooltip("Dying will delete your file.\n\n" ICON_FA_EXCLAMATION_TRIANGLE
                                            " WARNING " ICON_FA_EXCLAMATION_TRIANGLE
                                            "\nTHIS IS NOT REVERSIBLE!\nUSE AT YOUR OWN RISK!"));
+    AddWidget(path, "Fire Timer Multiplier", WIDGET_CVAR_SLIDER_INT)
+        .CVar(CVAR_ENHANCEMENT("FireTimerMultiplier"))
+        .Options(IntSliderOptions().Min(-5).Max(5).DefaultValue(0).Format("%+d").Tooltip("-5 will be half as much time, +5 will be 6x as much time"));
```

Comment:
Not sure if we want to special case hammer chest here, which quickly requires damage boosting past curtain since it's not really feasible to reach it much quicker